Classes/Service/ItemsProcFunc.php

```diff
@@ -81,4 +86,30 @@ protected function getScope(array $params)
         }
         return $scope;
     }
+
+    /**
+     * @param array  $tvpPageTsConfig
+     * @param string $mappingPlace
+     *
+     * @return bool
+     */
+    protected static function checkIfMapIsFiltered(array $tvpPageTsConfig, string $mappingPlace): bool
+    {
+        if (isset($tvpPageTsConfig['filterMaps.'])) {
+            $allowedPlaces = $tvpPageTsConfig['filterMaps.'];
+        }
+        elseif (isset($tvpPageTsConfig['filterMaps'])) {
```

Comment:
The elseif or else should go one line above

Reply:
It seems you suggest a specific code style I tried some but couldn't make my PHPStorm not change files so I'm just guessing. 
Should perhaps code style rules be added to the project? I could create a PR for the default TYPO3 core php-cs-fixer rules.